config/config.go

```diff
@@ -10,12 +10,15 @@ import (
 // Config is the filing resource handler config
 type Config struct {
 	BindAddr                  string        `envconfig:"BIND_ADDR"                  json:"-"`
+	Brokers                   []string      `envconfig:"KAFKA_ADDR"                 json:"-"`
 	DatasetAPIURL             string        `envconfig:"DATASET_API_URL"`
 	DatasetAPISecretKey       string        `envconfig:"DATASET_API_SECRET_KEY"     json:"-"`
 	ElasticSearchAPIURL       string        `envconfig:"ELASTIC_SEARCH_URL"         json:"-"`
 	GracefulShutdownTimeout   time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval       time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckTimeout        time.Duration `envconfig:"HEALTHCHECK_TIMEOUT"`
+	HierarchyBuiltTopic       string        `envconfig:"HIERARCHY_BUILT_TOPIC"`
+	KafkaMaxBytes             string        `envconfig:"KAFKA_MAX_BYTES"`
```

Comment:
why not an `int` or `int64`?

Reply:
To be consistent with other repositories, (filter API used string)

Reply:
I will change it and update dp-ci with the new configuration value